client/allocrunner/taskrunner/task_dir_hook.go

```diff
@@ -67,15 +67,20 @@ func (h *taskDirHook) Prestart(ctx context.Context, req *interfaces.TaskPrestart
 	}
 
 	// Update the environment variables based on the built task directory
-	setEnvvars(h.runner.envBuilder, fsi, h.runner.taskDir, h.runner.clientConfig)
+	SetEnvvars(h.runner.envBuilder, fsi, h.runner.taskDir, h.runner.clientConfig)
 	resp.State = map[string]string{
 		TaskDirHookIsDoneDataKey: "true",
 	}
 	return nil
 }
 
-// setEnvvars sets path and host env vars depending on the FS isolation used.
-func setEnvvars(envBuilder *taskenv.Builder, fsi drivers.FSIsolation, taskDir *allocdir.TaskDir, conf *cconfig.Config) {
+// SetEnvvars sets path and host env vars depending on the FS isolation used.
```

Comment:
this was redundant with 
https://github.com/hashicorp/nomad/pull/9668/files#diff-599b8e928ddf62defbba8253f2424a27f4defe9d9f2ad23fa66498e5f8c02477L252

combined them and exported